### ISSUE TITLE
--command option doesn't work as expected

### ISSUE THREAD

**Issue by fedosejev** [substack/catw#1]
For some reason ``` --command``` option doesn't work as expected in this case:
``` catw --command --verbose 'lessc --compress -' 'source/less/styles.less' > build/css/styles.css```

While ``` -c``` works fine:
``` catw -c --verbose 'lessc --compress -' 'source/less/styles.less' > build/css/styles.css```

Even though it does throw an error:
```javascript
/usr/local/lib/node_modules/catw/node_modules/shell-quote/index.js:57
    var match = filter(s.match(chunker), Boolean);
                         ^
TypeError: Object true has no method 'match'
    at parse (/usr/local/lib/node_modules/catw/node_modules/shell-quote/index.js:57:26)
    at exports.parse (/usr/local/lib/node_modules/catw/node_modules/shell-quote/index.js:37:18)
    at /usr/local/lib/node_modules/catw/bin/cmd.js:38:21
    at /usr/local/lib/node_modules/catw/index.js:105:20
    at Array.map (native)
    at /usr/local/lib/node_modules/catw/index.js:104:39
    at nextFile (/usr/local/lib/node_modules/catw/index.js:76:31)
    at /usr/local/lib/node_modules/catw/index.js:83:19
    at Glob.<anonymous> (/usr/local/lib/node_modules/catw/node_modules/glob/glob.js:96:7)
    at Glob.EventEmitter.emit (events.js:95:17)
```

Any hints on why is this happening?

**Comment by k88hudson** [comment]
my pull request should resolve this, it looks like command was not actually aliased 
https://github.com/substack/catw/pull/2

**Comment by substack** [comment]
Should be all fixed in 0.2.1/1.0.0.
